docs/policygenerator-reference.yaml

```diff
@@ -43,6 +43,10 @@ policyDefaults:
     # avoid evaluating the policy after it has become a particular compliance state.
     compliant: 30m
     noncompliant: 45s
+  # Optional. Determines whether objects created or monitored by the policy should be deleted when the policy is
```

Comment:
Not sure if the difference between "DeleteIfCreated", "DeleteAll" so be mentioned here, the rest looks good to me

Reply:
It's a good thought, but I think I'd prefer to keep these descriptions on the shorter side, as with `complianceType`. And then, as Matt pointed out below, if another value comes along we don't have to feel like we need to add it here.